packages/db-states-redis/src/lib/states/statesInRedisClient.ts

```diff
@@ -627,9 +718,13 @@ class StateRedisClient {
      *      <li><b>lc</b>   a unix timestamp indicating the last change of the actual value. this should be undefined
      *                      when calling setState, it will be set by the setValue method itself.</li></ul>
      *
-     * @param callback {function(Error|undefined, String):void}   will be called when redis confirmed reception of the command
+     * @param callback will be called when redis confirmed reception of the command
      */
-    async setState(id, state, callback) {
+    async setState(
+        id: string,
+        state: ioBroker.SettableState | any,
```

Comment:
`<anything> | any` is just `any`. Is this an `ioBroker.SettableState` or not?

Reply:
was more like an todo, yes will try to fix it